src/PGLib.jl

```diff
@@ -108,6 +108,29 @@ function _find_pglib_case(name::AbstractString, path::AbstractString)
     return filtered_files
 end
 
+"""
+    `nbuses(name::AbstractString)`
+
+    Return the number of buses, as indicated by the filename.
+"""
+function nbuses(name::AbstractString)
```

Comment:
This function probably needs a better name. Maybe `get_buscount`? Open to suggestions.

Reply:
I think `get_buscount` works better, but I am fairly impartial.   You can merge when ready.